dateutil/test/test_tz.py

```diff
@@ -1910,9 +1916,19 @@ def testFilestreamWithNameRepr(self):
 
     def testRoundNonFullMinutes(self):
         # This timezone has an offset of 5992 seconds in 1900-01-01.
-        tzc = tz.tzfile(BytesIO(base64.b64decode(EUROPE_HELSINKI)))
-        self.assertEqual(str(datetime(1900, 1, 1, 0, 0, tzinfo=tzc)),
-                             "1900-01-01 00:00:00+01:40")
+        # For python version pre-3.6, this will be rounded
+        set_as_old_python()
```

Comment:
There's no need to mock out the old behavior - we don't need to verify that this behavior is accurate on all versions of Python. It's best to split this into two tests, one for pre-3.6 and one for 3.6+, and then use `pytest.skipif` to run them selectively based on version.